qwandaq/src/main/java/life/genny/qwandaq/utils/DatabaseUtils.java

```diff
@@ -43,10 +43,11 @@ public class DatabaseUtils {
 	/**
 	 * Check if entityManager is present.
 	 */
-	public void checkEntityManager() {
+	public boolean checkEntityManager() {
 		if (entityManager == null) {
 			throw new NotInitializedException("EntityManager not initialized");
 		}
+		return entityManager != null;
```

Comment:
"true" could be returned

Reply:
Ah. This should not be here. I will update this :D